kubernetes_state/metadata.csv

```diff
@@ -98,3 +98,9 @@ kubernetes_state.statefulset.replicas_desired,gauge,,,,The number of desired rep
 kubernetes_state.statefulset.replicas_current,gauge,,,,The number of current replicas per StatefulSet,0,kubernetes,k8s_state.statefulset.replicas_current
 kubernetes_state.statefulset.replicas_ready,gauge,,,,The number of ready replicas per StatefulSet,0,kubernetes,k8s_state.statefulset.replicas_ready
 kubernetes_state.statefulset.replicas_updated,gauge,,,,The number of updated replicas per StatefulSet,0,kubernetes,k8s_state.statefulset.replicas_updated
+kubernetes_state.telemetry.payload.size,gauge,,,,The message size received from kube-state-metrics,0,kubernetes,k8s_state.telemetry.payload.size
```

Comment:
Is there a unit for `kubernetes_state.telemetry.payload.size` ? bytes ?

Reply:
yes bytes